resources/docs/source/includes/_structures/chat_channel_show.md

```diff
@@ -0,0 +1,53 @@
+## ChatChannelShow
```

Comment:
is it going to be used anywhere else? Seems rather weird separating it from the route itself